### DISCUSSION TITLE
link is not opening full pkg

### DISCUSSION THREAD

**Discussion by Tanveer-boat** [termux/termux-packages#10955] [Q&A] [answered]
Brother any help my termux is not showing link of all packages like - zphisher, saycheese, camhack. You will be greatly appreciated, tell me how to fix it ️😩

**Comment by sk0kanik** [comment, marked as answer]
"if you decide to use Termux **for hacking purposes**, you agree that you are on your own. Warranty is lost at this point. We will not assist you in fixing issues, whether or not related to your activity. If certain tools do not work or in any other way malfunctioning, report issues to their distributors or original authors.
Remember that Termux staff and community are not responsible for all your hacking troubles, lost data, broken devices and other damage caused by your usage of Termux." -  https://wiki.termux.com/wiki/Hacking
